test/e2e_test.go

```diff
@@ -1029,7 +1032,7 @@ func TestRekorBundle(t *testing.T) {
 
 	// Make sure offline verification works with bundling
 	// use rekor prod since we have hardcoded the public key
```

Comment:
Related to the above comment, it looks like the `verify` function sets `IgnoreTlog` to true, so it's not checking if the entry is in the log. This is fine, we can do a pass over the e2e tests later to make sure we're covering a few key use cases. We should be good to remove this comment and line though in each test.